source/extensions/common/wasm/wasm.h

```diff
@@ -370,6 +383,9 @@ class Wasm : public Envoy::Server::Wasm,
 
   std::unique_ptr<Global<double>> emscripten_NaN_;
   std::unique_ptr<Global<double>> emscripten_Infinity_;
+
+  WasmCall1Int _malloc_;
+  WasmCall1Void _free_;
```

Comment:
Nit: `s/_malloc_/malloc_/` `s/_free_/free_/` and move it after on `onTick_`?

Reply:
done